src/data_table/local_data_table.js

```diff
@@ -491,6 +539,9 @@ class LocalDataTable extends View {
       aoColumns: this._columnManager.dataTableColumnsConfig(),
       aaSorting: this._columnManager.dataTableSortingConfig(),
       fnDrawCallback: this._onDraw,
+      iDisplayStart: displayStart,
+      iRecordsTotal: recordTotal,
+      iRecordsDisplay: recordTotal,
```

Comment:
can you move these up next to `iDsplayLength` as they are related?

Reply:
✅